### ISSUE TITLE
Bug Report: In-Game Album Art For Vol. 1 Is Outdated

### ISSUE THREAD

**Issue by Emerald42069** [FunkinCrew/Funkin#3049]
### Issue Checklist

- [X] I have properly named the issue
- [ ] I have checked the issues/discussions pages to see if the issue has been previously reported

### What platform are you using?

Itch.io (Downloadable Build) - Windows

### If you are playing on a browser, which one are you using?

None

### Version

0.4.1

### Context (Provide images, videos, etc.)

In the current build of the game on the freeplay menu it uses the old album cover for Vol. 1 instead of the updated one 

_In-Game_
![IMG_2469](https://github.com/user-attachments/assets/ffc665e1-4920-425b-9cdd-ffc9bd30eee8)

_Intended Art_
![image](https://github.com/user-attachments/assets/297a6ec7-e0b1-4fe4-89e4-ff9df0829ba1)

### Steps to reproduce (or crash logs, errors, etc.)

Open Freeplay

**Comment by Emerald42069** [comment]
Wrong image 
![IMG_2470](https://github.com/user-attachments/assets/96d56b37-5243-462d-b3cc-7ad9a79fcae6)

**Comment by JVNpixels** [comment]
I have a feeling it would be redundant (repetitive) to have 2 of those albums in the game, but if your saying that it should be upgraded to the new Volume 1 album art then I agree

**Comment by Emerald42069** [comment]
Yes I was saying that it should be upgraded

**Comment by JVNpixels** [comment]
We can see what developers say about it, it may come in 0.5 or future updates. Same with OST expansion, there has also been a ton of reports (including some of my own) about the albums being outdated, unadded, incorrect, and etc.

**Comment by Emerald42069** [comment]
I added the new vol. 1  cover into the game heres how it looks
![screenshot](https://github.com/user-attachments/assets/939c3a7f-8a82-4fa4-8934-cce5a6fcbc84)
and heres the spritemap
![spritemap1](https://github.com/user-attachments/assets/822b2956-ccd9-46aa-9062-46f5ab5a2558)

**Comment by JVNpixels** [comment]
nice! if you want you can make a pull request on [Friday Night Funkin’s asset repository](https://github.com/FunkinCrew/funkin.assets), and fork the repository (there should be a fork button at the top of the page once you land there), go to your account and click repositories, and open your forked funkin.assets.

**Comment by JVNpixels** [comment]
then afterwards, find the destination path you changed those assets, like where ever you changed the album art, delete the old one and add your version, then afterwards there should be a green button to commit your changes, and press create a new pull request. fill it out like how you usually would with an issue, and then submit.

**Comment by EliteMasterEric** [comment]
The in-game album art is definitely correct.
